examples/string_token_substitution.ipynb

```diff
@@ -0,0 +1,247 @@
+{
+ "cells": [
+  {
+   "metadata": {},
+   "cell_type": "markdown",
+   "source": "# String token substitution",
+   "id": "4c4a5a5ed39eb2e1"
+  },
+  {
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-06-05T09:33:38.946449Z",
+     "start_time": "2024-06-05T09:33:38.943885Z"
+    }
+   },
+   "cell_type": "code",
+   "source": "from resources import helpers",
+   "id": "8b19d0e7242a0a53",
+   "outputs": [],
+   "execution_count": 7
```

Comment:
[Consider] Is it possible to have this lower in the doc? Just so that the actual introduction is the first thing the eye hits, and you only get code once samples start showing up?

Reply:
Good idea. I was wondering what to do with it.  Should have occurred to me to just move it down with the first bit of code.

Reply:
Moved down in f1bb7bc9